### PR TITLE
[8.0][NEW] project_task_analytic_partner: New module

### PR DESCRIPTION
**Pull request by carlosdauden**
This module adds the partner on task work lines and propagate it to the auto-generated analytic entries on the "Other partner" field.

### CODE REVIEW

File: project_task_analytic_partner/README.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+   :alt: License: AGPL-3
+
+=====================
+Partner in task works
+=====================
+
+This module adds the partner on task works and propagate it to the
+auto-generated analytic entries on the "Other partner" field.
+
+
```
Comment (pedrobaeza) [inline project_task_analytic_partner/README.rst:13]: Add Installation section mentioning the dependencies to OCA/account-analytic

File: project_task_analytic_partner/README.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+=============
+
+* Go to *Settings -> Configuration -> Project -> Project Management*.
+* Enable *Record timesheet lines per tasks*.
+
+
+Usage
+=====
+
+Go to Project > Tasks, and you will find the partner field in work lines.
```
Comment (pedrobaeza) [inline project_task_analytic_partner/README.rst]: Mention also about the purpose of this field.

**Comment by pedrobaeza** [comment]
Little comments, but :+1:

File: project_task_analytic_partner/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner in task works',
+    'summary': 'Select alternative partner in task works',
```
Comment (dreispt) [inline project_task_analytic_partner/__openerp__.py]: I think it should be "task work" or "task work lines"

File: project_task_analytic_partner/README.rst (hunk 1 above)
Comment (dreispt) [inline project_task_analytic_partner/README.rst:13]: AFAICS this module requires HR Timesheet to work.
This is not clear from the title, and should be explained here.

File: project_task_analytic_partner/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    'website': 'http://www.tecnativa.com',
+    'depends': [
+        'analytic_partner_hr_timesheet_invoice',
+        'project_timesheet',
+    ],
+    'data': [
+        'views/project_view.xml',
+    ],
+    'installable': True,
+    'autoinstall': True,
```
Comment (dreispt) [inline project_task_analytic_partner/__openerp__.py]: Why?

**Comment by dreispt** [comment]
The README needs some improvements.
Other than that it's OK.

**Comment by carlosdauden** [comment]
Changes done

File: project_task_analytic_partner/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.tecnativa.com',
+    'depends': [
+        'analytic_partner_hr_timesheet_invoice',
+        'project_timesheet',
+    ],
+    'data': [
+        'views/project_view.xml',
+    ],
+    'installable': True,
```
Comment (dreispt) [inline project_task_analytic_partner/__openerp__.py:21]: @pedrobaeza Should this one also be auto installable?

Comment (pedrobaeza) [inline project_task_analytic_partner/__openerp__.py:21]: Well, this is where you can have the control to enable/disable the feature. We can put it if we want, as the same explanation of the other applies also to this one. What you prefer.

Comment (dreispt) [inline project_task_analytic_partner/__openerp__.py:21]: No opinion here. I'm not familiar with the `analytic_partner_hr_timesheet_invoice` module or the related use case.
Whatever makes more sense to you. If in doubt, don't use autoinstall.

**Comment by rafaelbn** [comment]
Amazing, thanks! :+1: Tested in runbot!

**Pull request merged**